src/client/components/Appheader/AppHeader.component.js

```diff
@@ -0,0 +1,20 @@
+// dumb/statless img component with props
+
+import React from 'react';
+import './AppHeader.style.css';
+import PropTypes from 'prop-types';
+
+export const AppHeader = (props) => {
```

Comment:
you could also desctructure your `props`. so you don't have to write `props`.

```suggestion
export const AppHeader = ({ src }) => {
```